rustworkx/rustworkx.pyi

```diff
@@ -207,9 +207,11 @@ def graph_bipartite_edge_color(graph: PyGraph, /) -> dict[int, int]: ...
 
 def connected_components(graph: PyGraph, /) -> list[set[int]]: ...
 def is_connected(graph: PyGraph, /) -> bool: ...
+def is_strongly_connected(graph: PyDiGraph, /) -> bool: ...
 def is_weakly_connected(graph: PyDiGraph, /) -> bool: ...
 def is_semi_connected(graph: PyDiGraph, /) -> bool: ...
 def number_connected_components(graph: PyGraph, /) -> int: ...
+def number_strongly_connected_components(graph: PyDiGraph, /) -> bool: ...
```

Comment:
```suggestion
def number_strongly_connected_components(graph: PyDiGraph, /) -> int: ...
```

Reply:
Oh, I cloned the line below (`number_weakly_connected_components`) and it had also `bool`. Will fix both. Thanks.